code.gs

```diff
@@ -1876,6 +1876,37 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'נראה שאתה משתמש בגרסה לא עדכנית של התוכנה',
     'You can find the latest one here': 'אתה יכול להוריד את הגרסה העדכנית כאן',
   },
+  'ru': {
+    'Age': 'Возраст',
+    'Years': 'Лет',
+    'Events': 'События',
+    'Birthdays today': 'Дни рождения сегодня',
+    'Birthdays tomorrow': 'Дни рождения завтра',
+    'Birthdays in {0} days': 'Дни рождения через {0} дней',
+    'Anniversaries today': 'Юбилей сегодня',
+    'Anniversaries tomorrow': 'Юбилей завтра',
+    'Anniversaries in {0} days': 'Юбилей через {0} дней',
+    'Custom events today': 'Специальное событие сегодня',
+    'Custom events tomorrow': 'Специальное событие завтра',
+    'Custom events in {0} days': 'Специальное событие через {0} дней',
+    'Hey! Don\'t forget these events': 'Эй! Не забудь об этих мероприятиях',
+    'version': 'версия',
+    'dd-MM-yyyy': 'ДД-ММ-ГГГГ',
```

Comment:
Please see my comment in your previous PR and the note on line 1915.  
The string 'dd-MM-yyyy' should not be translated: just reordered.

Reply:
>.< sorry! Fixed